include/mbedtls/ssl_cache.h

```diff
@@ -132,9 +132,9 @@ int mbedtls_ssl_cache_set(void *data,
  *                        associated to \p session.
  * \param session_id_len  The length of \p session_id in bytes.
  *
- * \return                0: The cache entry for session with provided ID
+ * \return                \c 0: The cache entry for session with provided ID
  *                           is removed or does not exist.
```

Comment:
This should be aligned. 
```suggestion
 * \return                \c 0 on success. This indicates the cache entry for
 *                        session with provided ID is removed or does not exist.
```